src/pages/StudyHours.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import Footer from '../components/Footer';
+import placeholderPic from '../assets/placeholderPic.png';
+import studyHoursBackground from '../assets/StudyHoursBackground.png';
+import studyHoursBackground2 from '../assets/StudyHoursBackground2.png';
+import studyHoursSection1 from '../assets/StudyHoursSection1.png';
+import studyHoursSection2 from '../assets/StudyHoursSection2.png';
+import studyHoursSection3 from '../assets/StudyHoursSection3.png';
+import studyHoursSection1Image from '../assets/Section1Image.png';
+import studyHoursSection2Image from '../assets/Section2Image.png';
+import studyHoursSection3Image from '../assets/Section3Image.png';
+import StudyHoursTextWhiteBackground from '../assets/StudyHoursTextWhiteBackground.png'
+import StudyHoursTextGreyBackground from '../assets/StudyHoursTextGreyBackground.png'
+
+const StudyHours = () => {
+    return (
+        <>
+            <div className="min-h-screen bg-white">
+                <main>
+                    <section className="relative mb-16">
+                        <div className="relative text-center">
+                            <img src={studyHoursBackground} alt="Placeholder" className="mx-auto" />
+                            <div className="absolute inset-0 flex flex-col justify-center items-center">
+                                <p className="text-3xl text-white">PROGRAMS</p>
+                                <h2 className="text-7xl font-bold text-white">STUDY HOURS</h2>
+                            </div>
+                        </div>
+                    </section>
```

Comment:
There's a header component that exists that you can utilize, its called navbar in components